catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -1018,6 +1019,7 @@ private void alignStream(InputStream in, long requestedBytesToSkip, long respons
         throw new IOException("Server skipped more bytes than requested in the range header.");
       }
     } catch (IOException e) {
+      IOUtils.closeQuietly(in);
```

Comment:
We only want to close the stream if an exception is thrown?

Reply:
If the exception is not thrown the stream needs to be passed into the ResourceImpl constructor which assumes responsibility for closing it. This change only prevents the edge case of an exception being thrown and the stream staying open without being passed into that constructor.

Reply:
❓  Is that worth making a comment in the code, might help with developers who look at this down the road.

Reply:
I don't think it will hurt. Adding